src/pages/index.tsx

```diff
@@ -1,51 +1,32 @@
 import { type NextPage } from "next";
 import Head from "next/head";
-import Link from "next/link";
+import Card from "../components/card/Card";
```

Comment:
Two small suggestions :)

You could use [aliases](https://nextjs.org/docs/advanced-features/module-path-aliases) for this.

for example
```suggestion
import Card from "components/card/Card";
```
or
```suggestion
import Card from "@components/card/Card";
```

I understand that this project might not have a very large scope, but it's very good practice to simplify common folders imports.

You can even go further and add an index file to components/card and import it as: 
```suggestion
import { Card } from "@components/card";
```

Reply:
Yes, this is on my todo list to create some base urls to make importing and moving code easier

Reply:
If you have a written todo list somewhere, it'd be helpful to add it to README.md, so I don't point out stuff that's on the list. thanks!